### PR TITLE
Changes implicit nulls to be explicit

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

Replaces implicit nulls (like having a space between 2 commas in a proc call `, ,`) with `null`.

## Why It's Good For The Game

Code readability

## Testing

Compiled

## Changelog
NPFC

### CODE REVIEW

File: code/modules/instruments/songs/play_legacy.dm [hunk 1]
```diff
@@ -17,7 +17,7 @@
 			var/len = length(notes_tempodiv)
 			if(len >= 2)
 				tempodiv = text2num(notes_tempodiv[2])
-			if(len)			//some dunkass is going to do ,,,, to make 3 rests instead of ,/1 because there's no standardization so let's be prepared for that.
+			if(len)			//some dunkass is going to do , null, null, null, to make 3 rests instead of ,/1 because there's no standardization so let's be prepared for that.
```
Comment (Henri215) [inline code/modules/instruments/songs/play_legacy.dm:20]: What are these comments about?

Comment (Contrabang) [reply code/modules/instruments/songs/play_legacy.dm:20]: :clueless: <-- find and replaced

**Review by Henri215 — APPROVED** [review]
, , good, to, me

**Review by S34NW — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]
FIWB

**Pull request merged**